.github/workflows/update-docs-manual.yml

```diff
@@ -0,0 +1,154 @@
+name: Manual Update MeshLib Documentation Site
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      output_folder:
+        required: false
+        type: string
+        default: "MeshLib/dev"
+  # for test
+  pull_request:
+    branches: [ master ]
+    inputs:
+      output_folder:
+        required: false
+        type: string
+        default: "MeshLib/dev"
+  
+jobs:
+  # NOTE: all output values are strings
+  # use the ${{ needs.config.outputs.bool-param == 'true' }} construction to check boolean values
+  config:
+    uses: ./.github/workflows/config.yml
+    
+  debug-config:
+    needs: config
+    runs-on: ubuntu-latest
+    steps:
+      - name: Print config
+        run: |
+          echo << EOF
+          ${{ toJSON( needs.config.outputs ) }}
+          EOF
```

Comment:
Not actually required here; debug printing is useful for checking for PR labels.

Reply:
fixed